### PR TITLE
Added squashing docs

### LINKED ISSUE #243: Add SQUASHING.md to this repo
We don't have it here. Can basically grab from Ember and place into here. https://github.com/code-corps/code-corps-ember/blob/develop/docs/SQUASHING.md

Would be good to add with details specific to a fork. See discussion in #241.

### PR DESCRIPTION
**Pull request by rileytaylor**
# What's in this PR?

This adds squashing.md to the API docs folder. It is copied over from the ember repository with the addition of instructions for working from a fork since that varies slightly.
## References

Fixes #243

### CODE REVIEW

File: docs/SQUASHING.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+## Rebase
+
+Rather than merge develop into your branch, we prefer to rebase off of develop. Depending on how you are working with the repo, this step may be different.
+
+If you are working from a forked copy run:
+
+```shell
+git rebase -i develop
```
Comment (rileytaylor) [inline docs/SQUASHING.md]: Here we could also do `git rebase -i HEAD~X`, X being the number of commits that have been done. As far as I know, this is functionally the same, just fancier and more likely to inspire the admiration of other git users 😛

Comment (rileytaylor) [inline docs/SQUASHING.md]: Nevermind, I've answered this myself and will update the branch

**Review by joshsmith — COMMENTED** [review]
Is there an easy way here to find out the number of commits added on your branch?

**Comment by rileytaylor** [comment]
@JoshSmith Yes. `git log` functions like a summarized `git diff`. So you can do `git log develop..your-branch-here` for local copies on a feature branch, or `git log origin/develop..develop` for a fork. You _might_ need to do upstream/develop depending on how it was set up but I haven't tested that yet.

**Review by joshsmith — APPROVED** [review]

**Comment by joshsmith** [comment]
🎈

**Pull request merged**